### DISCUSSION TITLE
New to HFS

### DISCUSSION THREAD

**Discussion by Cvscott81** [rejetto/hfs#715] [Q&A]
Hi all. I apologize in advance, I'm out of my depth here. I'm trying to setup a simple server on my Windows 10 computer to share files with my android devices. I installed HFS, created an account and added (I think) a folder to the shared files section. When I go to the provided web address on my phone, it doesn't load. Am I missing steps to make this work?

Thanks.

**Comment by rejetto** [comment]
try providing a screenshot, or at least say what address you are using.
It should be a local address, valid only in your local network.

**Comment by Cvscott81** [comment]
Thanks for the reply. I was able to get it to work. I didn't realize it had
provided two local addresses. The other one worked great.

I am experiencing a separate issue, though. I decided to move this server
over to an old laptop I have to use as a dedicated server machine. It's an
older Win10 laptop that should suffice. However, the install is not going
as smoothly. When I run the executable HFS file, the commandline gives me
these errors.[image: hfs error.PNG]



Any suggestions?

Thanks!

**Comment by rejetto** [comment]
the way you attached the screenshot didn't work.
it's probably not support by github when replying via email.
try using the github web page.

**Comment by Cvscott81** [comment]
Sorry about that. Here you go.
![hfs error](https://github.com/user-attachments/assets/0839ad6b-a302-4671-a625-4ecdc2633b81)

**Comment by rejetto** [comment]
for legacy reasons, hfs.exe by default stores data inside the same folder.
this is not permitted by Windows under "program files".
Easy solution: move hfs outside of program-files.
Other solution: run hfs from program-files, but specify a different working directory using parameter --cwd .
other solution: run hfs with enough permissions to write to program-files (not advised).

**Comment by Cvscott81** [comment]
Yes! That was it. Thank you so much!
